examples/geoparquet/example.py

```diff
@@ -5,17 +5,22 @@
 
 .. code-block:: python
 
-   >>> import json, pprint, pyarrow.parquet
+   >>> import json, pprint, pyarrow.parquet as pq
    >>> pprint.pprint(json.loads(pq.read_schema("example.parquet").metadata[b"geo"]))
-   {'columns': {'geometry': {'crs': 'GEOGCRS["WGS 84",ENSEMBLE["World Geodetic '
+   {'columns': {'geometry': {'bbox': [-180.0,
+                                      -90.0,
+                                      180.00000000000006,
+                                      83.64513000000001],
+                             'crs': 'GEOGCRS["WGS 84",ENSEMBLE["World Geodetic '
                                     'System 1984 ensemble",MEMBER["World Geodetic '
                                     'System 1984 (Transit)"],MEMBER["World '
                                     'Geodetic System 1984 (G730)"],MEMBER["World '
                                     'Geodetic System 1984 (G873)"],MEMBER["World '
                                     'Geodetic System 1984 (G1150)"],MEMBER["World '
                                     'Geodetic System 1984 (G1674)"],MEMBER["World '
+                                    'Geodetic System 1984 (G1762)"],MEMBER["World '
                                     'Geodetic System 1984 '
-                                    '(G1762)"],ELLIPSOID["WGS '
+                                    '(G2139)"],ELLIPSOID["WGS '
```

Comment:
I don't know why this changed... I created a new environment to run this. Maybe a change in geopandas or proj?

Reply:
Yes, every PROJ update can come with an updated database, and thus can change this output. Especially since EPSG:4326 is an ensemble CRS, it is by nature evolving.

Reply:
And it seems specifically this database update added this additional entry to the ensemble: https://github.com/OSGeo/PROJ/commit/f1efc8a7e444c306a57acd53d8283586132b2c09